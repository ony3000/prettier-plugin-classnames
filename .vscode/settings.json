{
  "editor.defaultFormatter": "esbenp.prettier-vscode",
  "editor.formatOnSave": true,
  "eslint.workingDirectories": [{ "mode": "auto" }]
}
