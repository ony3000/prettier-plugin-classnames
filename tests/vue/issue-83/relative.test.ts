import { thisPlugin, testSnapshotEach } from '../../adaptor';
import { baseOptions } from '../../settings';
import { fixtures } from './fixtures';

const options = {
  ...baseOptions,
  plugins: [thisPlugin],
  parser: 'vue',
  endingPosition: 'relative',
};

testSnapshotEach(fixtures, options);
