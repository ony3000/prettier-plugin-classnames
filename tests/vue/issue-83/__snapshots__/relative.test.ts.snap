// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`'(1) If a template literal contains neâ€¦' > expectation 1`] = `
"//---------------------------------------------------------|
printWidth=60 (in snapshot)
<template>
  <div>
    <div
      :class="
        clsx('w-full items-center justify-center', {
          'nodrag cursor-text h-full overflow-hidden':
            isEditing,
          [\`line-clamp-\${lineClamp}\`]: !isEditing,
        })
      "
    >
      Hello
    </div>
  </div>
</template>
"
`;
